articles/libraries/auth0-android/save-and-refresh-tokens.md

```diff
@@ -17,11 +17,7 @@ When an authentication is performed with the `offline_access` <dfn data-key="sco
 
 [Auth0.Android](https://github.com/auth0/Auth0.Android) provides a utility class to streamline the process of storing and renewing credentials. You can access the `accessToken` or `idToken` properties from the [Credentials](https://github.com/auth0/Auth0.Android/blob/master/auth0/src/main/java/com/auth0/android/result/Credentials.java) instance. This is the preferred method to manage user credentials.
 
-First, add the library dependency to your build.gradle file:
-
-```gradle
-compile 'com.auth0.android:auth0:1.+'
-```
+<%= include('../../quickstart/native/android/_includes/_gradle.md') %>
```

Comment:
I'd remove this include and the previous dependency snippet as well. I'd just say something like 
>"Credential Managers are included as part of the Auth0.Android SDK. If this is not part of your dependencies yet, make sure to check {link to https://auth0.com/docs/libraries/auth0-android}".  

And BTW, I think we can also make use of this new include in the Installation section of that main SDK document. See https://auth0.com/docs/libraries/auth0-android#installation

Reply:
- Link added using markdown and following relative paths convention on contribution guide
- Previous content was replaced with the new `_gradle.md` without a title. The previous link to Gradle docs shouldn't be needed anymore and seems bigger than the intended scope.